Extension/src/LanguageServer/client.ts

```diff
@@ -2976,6 +3035,55 @@ export class DefaultClient implements Client {
         }
     }
 
+    public async handleGenerateDoxygenComment(line: number | undefined, column: number | undefined): Promise<void> {
+        const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+        if (!editor) {
+            return;
+        }
+        const params: GenerateDoxygenCommentParams = {
+            uri: editor.document.uri.toString(),
+            position: (line !== undefined && column !== undefined) ? new vscode.Position(line, column) : editor.selection.active,
+            isCodeAction: (line !== undefined && column !== undefined) ? true : false
+        };
+
+        if (line !== undefined && column !== undefined) {
+            params.position = new vscode.Position(line, column);
+            params.isCodeAction = true;
+        } else {
+            params.isCodeAction = false;
+        };
+
+        await this.awaitUntilLanguageClientReady();
+        const oldVersion: number | undefined = openFileVersions.get(params.uri);
+        const result: GenerateDoxygenCommentResult | undefined = await this.languageClient.sendRequest(GenerateDoxygenCommentRequest, params);
+        const newVersion: number | undefined = openFileVersions.get(params.uri);
+
+        if (oldVersion === undefined) {
```

Comment:
If you added a check for result === undefined that would seem to elminate the need to use all the result?.